examples/hello-world/step-by-step/cifar10/data/readme.md

```diff
@@ -0,0 +1,53 @@
+# Problem and Data
+
+For this tutorial, we will use the CIFAR10 dataset. 
+It has the classes: ‘airplane’, ‘automobile’, ‘bird’, ‘cat’, ‘deer’, ‘dog’, ‘frog’, ‘horse’, ‘ship’, ‘truck’. 
+The images in CIFAR-10 are of size 3x32x32, i.e. 3-channel color images of 32x32 pixels in size.
+
+The pytorch tutorial will training an image classifier. The examples shows the following steps in order:
+
+* Load and normalize the CIFAR10 training and test datasets using torchvision
+* Define a Convolutional Neural Network
+* Define a loss function
+* Train the network on the training data
+* Test the network on the test data
+
+![image](./cifar10.png)
+
+
+# PREPARE DATA 
+
+Before we start the working on the problem, let's first prepare the data, assuming the dataset is already available, 
```

Comment:
No need for all caps, and maybe remove "Before we start working... two steps to prepare data"